frontend/src/pages/question/panels/NodePanel.jsx

```diff
@@ -17,6 +17,22 @@ import biolinkUtils from '@/utils/biolink';
 
 import CurieConceptSelector from '@/components/shared/curies/CurieConceptSelector';
 
+function normalizeNodeType(type) {
```

Comment:
I think there's probably a better name than normalize. Especially because biolink is switching to use "biolink:Gene" as the "normal" name, and it's really robokop that uses the "abnormal" formatting.

Reply:
I changed the function name and added some documentation. I also looked at the biolink model yaml file and the types haven't changed, my thought is that the node normalizer service is adding the biolink part to show where it was getting its types and we shouldn't be keeping that info. And I also think KGs are not going to have those types but just have 'disease'. Though I don't really understand why the types need this prefix. @patrickkwang do you have any insight?

Reply:
My feeling is that the biolink model itself is the only place that should _not_ use the `biolink:` prefix. Think of it like a namespace. KGs, KPs, ARAs, etc. will all use the full CURIE form.

Reply:
Ok, so I think we should still convert the node types to what we already expect internally. This will allow us to continue to send valid queries to Messenger and receive answers back. Down the road, when we add new ARAs or Messenger gets updated, then we can remove this conversion.

I think that there are other things we can take care of in the mean time to make node types more dynamic, like #146 so that when we remove the conversion, everything will still work.